docs/concepts/crds/repository.md

```diff
@@ -16,205 +16,122 @@ section_menu_id: concepts
+- **status.firstBackupTime**
+`status.firstBackupTime` indicates the timestamp when the first backup was taken.
 
-```
-# List all Repositories of all namespaces in the cluster
-$ kubectl get repository --all-namespaces
+- **status.lastBackupTime**
+`status.lastBackupTime` indicates the timestamp when the latest backup was taken.
 
-# List all Repositories created for a particular Restic
-$ kubectl get repository -l restic=stash-demo --all-namespaces
+- **status.integrity**
+Stash checks the integrity of backed up files after each backup. `status.integrity` shows the result of the integrity check.
 
-# List all Repositories of Deployment workloads of all namespaces
-$ kubectl get repository -l workload-kind=Deployment --all-namespaces
+- **status.size**
+`status.size` shows the total size of repository after last backup.
 
-# List all Respositories of a particular Workload
-$ kubectl get repository -l workload-kind=StatefulSet,workload-name=stash-demo
+- **status.snapshotCount**
+`status.SnapshotCount` shows the number of snapshots stored in the Repository.
 
-# List all Repositories of a particular node (DaemonSet only)
-$ kubectl get repository -l node-name=minikube
-```
+- **status.snapshotRemovedOnLastCleanup**
+`status.snapshotRemovedOnLastCleanup` shows the number of old snapshots that has been cleaned up according to retention policy on last backup session.
 
 ## Deleting Repository
 
-Stash allows the users to delete **only `Repository` crd** or **`Repository` crd with respective restic repository**. Here, we are going to show how to perform these delete operations.
+Stash allows the users to delete **only `Repository` crd** or **`Repository` crd along with respective backed up data**. Here, we are going to show how to perform these delete operations.
 
-### Delete only Repository crd
+**Delete only `Repository` keeping backed up data :**
 
  You can delete only `Repository` crd by,
 
 ```console
 $ kubectl delete repository <repository-name>
 
 # Example
-$ kubectl delete repository deployment.stash-demo
-repository "deployment.stash-demo" deleted
+$ kubectl delete repository gcs-demo-repo
+repository "gcs-demo-repo" deleted
 ```
 
-This will delete only `Repository` crd. It won't delete any backed up data from respective restic repository.
-
->If you delete `Repository` crd while stash-sidecar still exist on the workload, Stash will re-create the `Repository` crd and continue to take backup. In this case, `status` field of `Repository` crd will be reset.
-
-> If you don't want stash to re-create `Repository` crd, you have to stop Stash from taking backup. To see how to stop stash from taking backup see [here](/docs/guides/v1alpha1/backup.md#disable-backup).
+This will delete only `Repository` crd. It won't delete any backed up data from the backend.
 
-### Delete respective restic repository
+>If you delete `Repository` crd while respective stash sidecar still exists on the workload, it will fail to take further backup.
```

Comment:
WHat exactly happens? Does it log like crazy? Does it crashloop?

Reply:
- workload should not crashloop .
- Should work like normal once Repository is re-created.